README.md

```diff
@@ -5,7 +5,8 @@
 <img src="/images/1.png" height="400em" /> <img src="/images/2.png" height="400em" />
 <img src="/images/3.png" height="400em" /> <img src="/images/4.png" height="400em" />
 <img src="/images/5.png" height="400em" /> <img src="/images/6.png" height="400em" />
-<img src="/images/7.png" height="400em" />
+<img src="/images/7.png" height="400em" /> <img src="/images/8.jpg" height="400em" />
```

Comment:
could you remove jpg file from here i think only gif is perfect and looks good.

Reply:
just a min...